### DISCUSSION TITLE
Auto fix eslint errors on save

### DISCUSSION THREAD

**Discussion by hhosseinipaymentshield** [quasarframework/quasar#11694] [General - Components / Directives / etc]
How can I configure Quasar to fix eslint errors on file save?

**Comment by pdanpdan** [comment]
you configure your editor, not quasar :)

**Reply by hhosseinipaymentshield** [reply]
You can run `npm run lint -- --fix` to fix some of the errors, can you not have this enabled somehow on file save?

**Reply by ivanq3w** [reply]
If youre using vscode, you can create .vscode folder and settings.json file there with this:
```
"editor.codeActionsOnSave": {
  "source.fixAll.eslint": true,
},
```
And maybe you should have ESLint extension :)

**Reply by hhosseinipaymentshield** [reply]
Thanks. It's actually not for me, it's for a colleague of mine. I think he already has eslint extension installed. I was just wondering if you could do it without configuring our editors.
Thanks.

**Reply by ivanq3w** [reply]
We can configure webpack !)
https://webpack.js.org/plugins/eslint-webpack-plugin/
There is a fix option that lints and make a changes to source code.

**Reply by hhosseinipaymentshield** [reply]
Thanks, this is the sort of thing I was looking for. I will see if I can get it to work with Quasar.

**Reply by ivanq3w** [reply]
Mine quasar.config.js looks like this, but seems it is not working :(
Maybe the answer is here: https://eslint.vuejs.org/user-guide/#usage
![image](https://user-images.githubusercontent.com/58051684/146336693-b7ebda16-c641-487d-8ed8-c4e2a81913de.png)

**Reply by hhosseinipaymentshield** [reply]
I think I have already tried this and didn't work for me as well. Let me know please if you found a way.
